units/util.go

```diff
@@ -29,24 +30,30 @@ func HandlePoisonedHost(ctx context.Context, env evergreen.Environment, h *host.
 			}
```

Comment:
GitHub doesn't let me comment on L21, but if the host that timed out during `ps` is a docker container, is it supposed to decommission all containers in the parent?

Reply:
This is a good question. I'm not sure. 
If it happens in one container maybe it would happen in other existing and/or new containers?

Reply:
We don't have any insight from this PR as to how often or which hosts/distros have this ps issue so I don't know if it affects container distros. I'm going to assume it doesn't matter that much, so I'm fine with this.